checkov/common/util/http_utils.py

```diff
@@ -204,36 +206,54 @@ def request_wrapper(
 
 
 async def aiohttp_client_session_wrapper(
-        url: str,
+        method: str,
+        url: StrOrURL,
         headers: dict[str, Any],
-        payload: dict[str, Any]
-) -> int:
+        payload: dict[str, Any] | None = None,
+) -> tuple[int, Any]:
     request_max_tries = int(os.getenv('REQUEST_MAX_TRIES', 3))
     sleep_between_request_tries = float(os.getenv('SLEEP_BETWEEN_REQUEST_TRIES', 1))
 
+    # 1. Read proxy URL (may include user:pass for authentication)
+    proxy_url = os.environ.get("HTTPS_PROXY", os.environ.get("HTTP_PROXY"))
```

Comment:
those values can be 2 different fields, you don't have to use the same proxy for both http and https